pkg/ui/console/client.go

```diff
@@ -540,14 +540,37 @@ func printPolicyResponse(results []*policy.ExecutionResult) {
 			}
 		}
 
+		maxNameLength := 0
+		maxDescrLength := 0
+		for _, res := range execResult.Results {
+			if len(res.Name) > maxNameLength {
+				maxNameLength = len(res.Name) + 1
+			}
+			if len(res.Description) > maxDescrLength {
+				maxDescrLength = len(res.Description) + 1
+			}
+		}
+
+		fmtString := fmt.Sprintf("\t%%s  %%-%ds %%-%ds %%%ds\n", maxNameLength, maxDescrLength, 10)
+
 		for _, res := range execResult.Results {
 			switch {
 			case res.Passed:
-				ui.ColorizedOutput(ui.ColorInfo, "\t%s  %-10s %-120s %10s\n", emojiStatus[ui.StatusOK], res.Name, res.Description, color.GreenString("passed"))
+				ui.ColorizedOutput(ui.ColorInfo, fmtString, emojiStatus[ui.StatusOK], res.Name, res.Description, color.GreenString("passed"))
 			case res.Type == policy.ManualQuery:
-				ui.ColorizedOutput(ui.ColorInfo, "\t%s  %-10s %-120s %10s\n", emojiStatus[ui.StatusWarn], res.Name, res.Description, color.YellowString("manual"))
+				ui.ColorizedOutput(ui.ColorInfo, fmtString, emojiStatus[ui.StatusWarn], res.Name, res.Description, color.YellowString("manual"))
 			default:
-				ui.ColorizedOutput(ui.ColorInfo, "\t%s %-10s %-120s %10s\n", emojiStatus[ui.StatusError], res.Name, res.Description, color.RedString("failed"))
+				ui.ColorizedOutput(ui.ColorInfo, fmtString, emojiStatus[ui.StatusError], res.Name, res.Description, color.RedString("failed"))
+				ui.ColorizedOutput(ui.ColorWarning, "\n")
+				// specific columns can be decided upon later
+				for index, column := range res.Columns {
+					if column == "cq_output" || column == "arn" || column == "id" {
+						for _, row := range res.Data {
+							ui.ColorizedOutput(ui.ColorInfo, "\t\t%s  %-10s \n", emojiStatus[ui.StatusError], fmt.Sprintf("%v", row[index]))
+							ui.ColorizedOutput(ui.ColorWarning, "\n")
```

Comment:
would move this print to its own function i.e printOffendingResources ? maybe a better name